release/serve_tests/workloads/single_deployment_1k_noop_replica.py

```diff
@@ -36,21 +36,23 @@
 # TODO:(jiaodong) We should investigate and change this back to 1k
 # for now, we won't get valid latency numbers from wrk at 1k replica
 # likely due to request timeout.
-DEFAULT_FULL_TEST_NUM_REPLICA = 100
+DEFAULT_FULL_TEST_NUM_REPLICA = 1000
 DEFAULT_FULL_TEST_NUM_TRIALS = 10
 
 # Cluster setup configs
-NUM_REDIS_SHARDS = 1
-REDIS_MAX_MEMORY = 10**8
-OBJECT_STORE_MEMORY = 10**8
 NUM_CPU_PER_NODE = 8
 
 # Deployment configs
 DEFAULT_MAX_BATCH_SIZE = 16
 
 # Experiment configs - wrk specific
 DEFAULT_SMOKE_TEST_TRIAL_LENGTH = "10s"
-DEFAULT_FULL_TEST_TRIAL_LENGTH = "10m"
+DEFAULT_FULL_TEST_TRIAL_LENGTH = "1m"
+
+# wrk config.
+# For more discussion, see https://github.com/wg/wrk/issues/205
+WRK_NUM_CONNECTIONS = 100
+WRK_NUM_THREADS = 10
```

Comment:
I don't think the number of CPUs matters too much given that these threads will be doing almost entirely IO. I am just using 100 & 10 because that's what we had in the microbenchmark to saturate a single HTTP proxy. We can tune further if need be.